cmd/uhc/account/users/cmd.go

```diff
@@ -0,0 +1,238 @@
+/*
+Copyright (c) 2019 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package users
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/openshift-online/uhc-cli/pkg/config"
+	"github.com/openshift-online/uhc-cli/pkg/util"
+	"github.com/openshift-online/uhc-sdk-go/pkg/client"
+	amv1 "github.com/openshift-online/uhc-sdk-go/pkg/client/accountsmgmt/v1"
+
+	"github.com/spf13/cobra"
+)
+
+var args struct {
+	debug bool
+	org   string
+}
+
+var Cmd = &cobra.Command{
+	Use:   "users",
+	Short: "Retrieve users and their roles",
+	Long:  "Retrieve information of all users/roles in the same organization",
+	Run:   run,
+}
+
+func init() {
+	// Add flags to rootCmd:
+	flags := Cmd.Flags()
+	flags.BoolVar(
+		&args.debug,
+		"debug",
+		false,
+		"Enable debug mode.",
+	)
+	flags.StringVar(
+		&args.org,
+		"org",
+		"", // Default value gets assigned later as connection is needed.
+		"Organization ID, deafaults to current users organization.",
```

Comment:
There is a typo in `deafaults`. And a period is more grammatically correct here: `Organization identifier. Defaults to the organization of the current user.`